test/runtests.jl

```diff
@@ -36,8 +28,9 @@ end
     @test Showoff.format_fixed_scientific(-Inf, 1, false) == "-∞"
     @test Showoff.format_fixed_scientific(NaN, 1, false) == "NaN"
     @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.34568×10⁻³"
```

Comment:
This test is failing, as I have still not implemented the engineering flag. Once the rest is approved, I will make sure this test passes, too.